bgpd/bgp_route.c

```diff
@@ -17807,6 +17852,148 @@ void bgp_config_write_distance(struct vty *vty, struct bgp *bgp, afi_t afi,
 	}
 }
 
+/* Show BGP path info hash keys */
```

Comment:
I don't understand the purpose of this new DEFUN.  Can you elaborate to me what this is needed and why it's different than just showing the prefix with it's paths?